### ISSUE TITLE
Document the configuration file

### ISSUE THREAD

**Issue by itowlson** [bytecodealliance/wasm-pkg-tools#70]
I understand that it's not stabilised yet, so maybe this is premature.  But writing up the location and schema in the README, or even in code comments, would save time scrobbling around in the [unit tests](https://github.com/bytecodealliance/wasm-pkg-tools/blob/9ba1112d6a441ca5e64db84438f32d1fb994671a/crates/wasm-pkg-common/src/config/toml.rs#L103)... _grin_

**Comment by thomastaylor312** [comment]
Yeah I just added some more in #68 for easier scrobbling, but we actually explicitly called this out in the last sig packaging call. I'll be shifting my attention to this once we get the cargo component work ready
